mobileapp/assets/book.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+
<!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
+<svg width="800px" height="800px" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Comment:
This comment is useless

Reply:
I don't understand why we need SVG and components ? I don't see these assets used elsewhere in the app 🧐